### ISSUE TITLE
Can't resolve './build/src/index.js'

### ISSUE THREAD

**Issue by ovensi** [shrhdk/text-to-svg#44]
I can't run v3.1.4 ,` Can't resolve './build/src/index.js'`.
I found your `index.js`file
```js
/**
 * Copyright (c) 2016 Hideki Shiro
 */

module.exports = require('./build/src/index.js');
```
And I can't found the folder named `build`in 3.1.4.
Maybe you should change it to
```js
module.exports = require('./src/index.js');
```

**Comment by mambahao** [comment]
+1

**Comment by fxa76** [comment]
just in case you are stuck with this here is what worked for me : 
1. Git clone the repository
2. npm install
3 npm run-script build 
4. copy the build directory in the node_modules/text-to-svg of your project

**Comment by shrhdk** [comment]
@ovensi and everyone.

Thank you for reporting, and I'm sorry for the problem.
I am working on fix this issue.

**Comment by shrhdk** [comment]
I released v3.1.5 including `build/src/index.js`. Could you try this version?
